src/Sulu/Bundle/AdminBundle/Resources/js/stores/MultiSelectionStore/MultiSelectionStore.js

```diff
@@ -27,10 +27,20 @@ export default class MultiSelectionStore<T = string | number, U: {id: T} = Objec
         this.loadItems(selectedItemIds);
     }
 
+    @computed get ids(): Array<T> {
+        // TODO use metadata instead of hardcoded id
+        return this.items.map((item) => item.id);
+    }
+
     @action set(items: Array<U>) {
         this.items = items;
     }
 
+    getById(id: T) {
```

Comment:
is it possible that you are not using this anywhere?

Reply:
Seems like it got lost in another refactoring as well... Can remove it again.